tests/test_charm_builder.py

```diff
@@ -1223,7 +1223,7 @@ def test_processrun_crashed(tmp_path):
 
     # get a real exception to build the message as its internal text varies across OSes
     try:
-        subprocess.run([nonexistent])
+        subprocess.run([nonexistent], check=True)
```

Comment:
The subprocess.run changes were manual, just adding `check=True` when there was no check parameter.